test/test_files/tinysnb/uuid/csv.test

```diff
@@ -0,0 +1,21 @@
+-GROUP UUIDCSV
```

Comment:
Instead of creating a new testfile, you can use the existing datase: tinysnb.
You can randomly choose a node and rel csv file, and add a new uuid column to those csv files. We have more tests coverage on the tinysnb dataset.